playbooks/init/basic_facts.yml

```diff
@@ -82,18 +82,28 @@
       tasks_from: check_existing_config.yml
 
   - set_fact:
-      osm_cluster_network_cidr: "{{ l_existing_clusternet_cidr }}"
-      osm_host_subnet_length: "{{ l_existing_clusternet_len }}"
       openshift_portal_net: "{{ l_existing_config_master_config.networkConfig.serviceNetworkCIDR }}"
-    vars:
-      # networkConfig.clusterNetworks may not be present in upgraded clusters
-      l_clusternet_list: "{{ l_existing_config_master_config.networkConfig.clusterNetworks is defined | bool }}"
-      l_existing_clusternet_cidr: "{{ l_clusternet_list | ternary((l_existing_config_master_config.networkConfig.clusterNetworks[0].cidr | default('')), l_existing_config_master_config.networkConfig.clusterNetworkCIDR) }}"
-      l_existing_clusternet_len: "{{ l_clusternet_list | ternary((l_existing_config_master_config.networkConfig.clusterNetworks[0].hostSubnetLength | default('')), l_existing_config_master_config.networkConfig.hostSubnetLength) }}"
     when:
     - l_existing_config_master_config is defined
     - l_existing_config_master_config.networkConfig is defined
 
+  - set_fact:
```

Comment:
Perhaps these tasks should be in a block with 
```
when:
    - l_existing_config_master_config is defined
    - l_existing_config_master_config.networkConfig is defined
```
and additional `when`'s to improve readability?